src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -84,6 +85,10 @@ public CommandResult execute(Model model) throws CommandException {
         Person personToEdit = lastShownList.get(index.getZeroBased());
         Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
 
+        if (!personToEdit.getReminders().isEmpty() && editPersonDescriptor.isNameEdited()) {
```

Comment:
Good change to check whether person to edit has a reminder and whether the person's name is edited